jujugui/static/gui/src/app/components/entity-details/test-entity-details.js

```diff
@@ -28,7 +28,7 @@ describe('EntityDetails', function() {
 
   beforeAll(function(done) {
     // By loading these files it makes their classes available in the tests.
-    YUI().use('entity-details', function() { done(); });
+    YUI().use('entity-details', 'jujulib-utils', function() { done(); });
```

Comment:
this shouldn't be required as it should be included from the requires in the `entity-details.js` file. Were you seeing otherwise?